Makefile

```diff
@@ -1,7 +1,32 @@
+REGISTRY?=kubernetes-incubator
+IMAGE?=k8s-custom-metric-adapter-sample
+TEMP_DIR:=$(shell mktemp -d)
 ARCH?=amd64
+ALL_ARCH=amd64 arm arm64 ppc64le s390x
+ML_PLATFORMS=linux/amd64,linux/arm,linux/arm64,linux/ppc64le,linux/s390x
 OUT_DIR?=./_output
 
-.PHONY: all test verify-gofmt gofmt verify
+VERSION?=latest
+GOIMAGE=golang:1.10
+
+ifeq ($(ARCH),amd64)
+	BASEIMAGE?=busybox
+endif
+ifeq ($(ARCH),arm)
+	BASEIMAGE?=armhf/busybox
+endif
+ifeq ($(ARCH),arm64)
+	BASEIMAGE?=aarch64/busybox
+endif
+ifeq ($(ARCH),ppc64le)
+	BASEIMAGE?=ppc64le/busybox
+endif
+ifeq ($(ARCH),s390x)
+	BASEIMAGE?=s390x/busybox
+	GOIMAGE=s390x/golang:1.10
+endif
```

Comment:
For this section, I'd prefer to build from `SCRATCH` and bypass this entirely.

Reply:
I need to fix that in the other repos I maintain.  We'll need to also mount in `/tmp` as an emptydir in the deployment as well.